python/ray/rllib/pg/pg.py

```diff
@@ -0,0 +1,85 @@
+import numpy as np
+
+import ray
+from ray.rllib.optimizers import LocalSyncOptimizer
+from ray.rllib.pg.pg_evaluator import PGEvaluator, RemotePGEvaluator
+from ray.rllib.agent import Agent
+from ray.tune.result import TrainingResult
+
+DEFAULT_CONFIG = {
+    # Number of workers (excluding master)
+    "num_workers": 4,
+    # Size of rollout batch
+    "batch_size": 512,
+    # Use LSTM model - only applicable for image states
+    "observation_filter": "NoFilter",
+    # Which reward filter to apply to the reward
+    "reward_filter": "NoFilter",
+    # Discount factor of MDP
+    "gamma": 0.99,
+    # GAE(gamma) parameter
+    "lambda": 1.0,
+    # Max global norm for each gradient calculated by worker
+    "grad_clip": 10.0,
```

Comment:
Shall we drop the gae / filter / clipping stuff to simplify the example?

Reply:
yes I think that's a really good idea; @alvkao58 what are your thoughts?

Reply:
Sure, that sounds good, I think I'll also drop the stuff relevant to the value function for sake of simplicity.